ansible_ai_connect/ai/api/telemetry/schema2.py

```diff
@@ -82,3 +83,21 @@ class AnalyticsProductFeedback:
     timestamp: str = field(
         default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
     )
+
+
+@frozen
+class OneClickTrialPlan:
+    created_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    expired_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    is_active: bool = False
```

Comment:
@goneri I am not sure how this boolean field works. When does its value become `False`, in which situation/s? Do we really need to export it to analytics / telesense?  Thanks!

Reply:
When the plan becomes inactive, we don't remove the association but the `is_active` flag changes from `True` to `False`.

We don't remove the user / plan relationship to be sure the user doesn't start yet another trial.

Reply:
I see two options:
- either you can skip the Trial plan when is_active is false, but you will have a hard time later telling how many users have gone through a Trial. This would also create a different between the event structure and our DB structure.
- or we can be consistent and returns the flatten information and just add a filter on the analytics side.